packages/transport/src/courier/index.ts

```diff
@@ -33,8 +33,6 @@ export class CourierTransport extends Transport {
       userSignature: options.userSignature,
     });
 
-    this.ws.connect();
```

Comment:
we handle connecting in a useEffect inside useTransport now.

this means if someone uses the transport without the hook they will manually need to connect